sdcm/nemesis.py

```diff
@@ -984,7 +963,12 @@ def disrupt_soft_reboot_node(self):
 
     @decorate_with_context(ignore_ycsb_connection_refused)
     def disrupt_rolling_restart_cluster(self, random_order=False):
-        self.cluster.restart_scylla(random_order=random_order)
+        try:
+            for node in self.cluster.nodes:
```

Comment:
it's not the only place in SCT we have a rolling restart

Reply:
- _enable_disable_table_encryption
- disrupt_mgmt_restore

Reply:
maybe we should consider moving this logic into `restart_scylla`

Reply:
node.restart_scylla and cluster.restart_scylla is used in many places(test_functional.py, mng_cli_test.py, audit.py,  artifacts_test.py, stop_compaction_test.py). But the lock mechanism is needed only when we have nemeses(in longevity tests). So let's keep it for nemeses only(at least for now)

will update _enable_disable_table_encryption and disrupt_mgmt_restore